components/camel-amqp/src/main/java/org/apache/camel/component/amqp/AMQPConnectionDetails.java

```diff
@@ -28,13 +28,19 @@
     public static final String AMQP_USERNAME = "AMQP_SERVICE_USERNAME";
 
     public static final String AMQP_PASSWORD = "AMQP_SERVICE_PASSWORD";
+    
+    private static final String KEYSTORE = "./src/test/resources/broker.ks";
+
+    private static final String TRUSTSTORE = "./src/test/resources/broker.ks";
```

Comment:
Why are you pointing to test folder? I guess you need to define a different default value for these field